### ISSUE TITLE
Could you please tell me why clip is used to calculate gram matrix instead of VGG.

### ISSUE THREAD

**Issue by Jamie-Cheung** [vvictoryuki/FreeDoM#12]
Your work is very impressive and interests me very much. However, I would like to ask about the description of gram in the paper. Could you please tell me why clip is used to get the vector and calculate gram matrix instead of vgg.

**Comment by HolmesShuan** [comment]
I believe CLIP may be more robust to images with artifacts generated by diffusion models compared to VGG, which is pre-trained on natural images. CLIP is trained on a larger dataset and leverages its text embedding space, rather than relying solely on category labels. However, this hypothesis deserves further experimentation.

**Comment by HolmesShuan** [comment]
![image](https://github.com/user-attachments/assets/b630dc21-dace-4b1a-acbd-e9fb39b65f8a)


> I believe CLIP may be more robust to images with artifacts generated by diffusion models compared to VGG, which is pre-trained on natural images. CLIP is trained on a larger dataset and leverages its text embedding space, rather than relying solely on category labels. However, this hypothesis deserves further experimentation.

**Comment by HolmesShuan** [comment]
> I believe CLIP may be more robust to images with artifacts generated by diffusion models compared to VGG, which is pre-trained on natural images. CLIP is trained on a larger dataset and leverages its text embedding space, rather than relying solely on category labels. However, this hypothesis deserves further experimentation.

![image](https://github.com/user-attachments/assets/b425ff2b-904f-4b94-912e-cf6eace039da)
